js/index.js

```diff
@@ -39,117 +44,187 @@ class Snake extends Grid {
     }
 
     #start() {
+        this.#controls.speed.setAttribute('disabled', 'disabled');
+        this.#controls.mode.setAttribute('disabled', 'disabled');
```

Comment:
this should be put in a separate method. For example "disableControls"
```suggestion
        #disableControls() {
            this.#controls.speed.setAttribute('disabled', 'disabled');
            this.#controls.mode.setAttribute('disabled', 'disabled');
        }
        
        #start() {
            this.#disableControls();
```